src/components/input/range/index.tsx

```diff
@@ -1,22 +1,36 @@
-import { Component, Event, EventEmitter, h, Host, Prop, Watch } from "@stencil/core"
+import { Component, Event, EventEmitter, h, Host, Method, Prop, Watch } from "@stencil/core"
+import { Color } from "../../../model"
+import { Clearable } from "../Clearable"
+import { Input } from "../Input"
+import { Looks } from "../Looks"
 
 @Component({
 	tag: "smoothly-input-range",
 	styleUrl: "style.css",
-	shadow: true,
+	scoped: true,
 })
-export class SmoothlyInputRange {
-	@Prop({ mutable: true }) value = 0
+export class SmoothlyInputRange implements Input, Clearable {
+	@Prop({ mutable: true, reflect: true }) value: number | undefined = undefined
+	@Prop({ reflect: true, mutable: true }) color?: Color
+	@Prop({ reflect: true, mutable: true }) looks: Looks = "plain"
 	@Prop() min = 0
 	@Prop() max = 100
 	@Prop() name: string
 	@Prop() labelText?: string
 	@Prop() step: number | "any" = "any"
+	@Event() smoothlyInputLooks: EventEmitter<(looks: Looks, color: Color) => void>
 	@Event() smoothlyInput: EventEmitter<Record<string, any>>
+	@Method()
+	async clear(): Promise<void> {
+		this.value = undefined
+	}
 	@Watch("value")
 	valueChanged(): void {
 		this.smoothlyInput.emit({ [this.name]: this.value })
 	}
+	componentWillLoad() {
```

Comment:
Move thus functions above all the `@Method` but bellow the `@Event` declarations and add return type.